spec/keycard/request/direct_attributes_spec.rb

```diff
@@ -3,31 +3,31 @@
 require "keycard/request/direct_attributes"
 
 RSpec.describe Keycard::Request::DirectAttributes do
-  let(:addr)    { '10.0.0.1' }
-  let(:base)    { double('request', env: { 'REMOTE_USER' => 'user', 'REMOTE_ADDR' => addr }) }
-  let(:request) { described_class.new(base) }
+  let(:addr) { '10.0.0.1' }
+  let(:rack_request) { double(:rack_request, env: { 'REMOTE_USER' => 'user', 'REMOTE_ADDR' => addr }) }
+  let(:attributes) { described_class.new(rack_request) }
```

Comment:
Great catch -- this naming was from an earlier concept of wrapping requests. Thanks!